pkg/pointerdb/audit/audit_segment.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+
+	"github.com/vivint/infectious"
+
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/paths"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/storage/meta"
+)
+
+// Audit to audit segments
+type Audit struct {
+	pointers pdbclient.Client
+	r        *rand.Rand
+}
+
+// NewAudit creates a new instance of audit
+func NewAudit(pointers pdbclient.Client, r *rand.Rand) *Audit {
+	return &Audit{
+		pointers: pointers,
+		r:        r,
+	}
+}
+
+// Stripe is a struct that contains stripe info
+type Stripe struct {
+	stripe int
```

Comment:
I think you may need to return more information here.

Reply:
also make them public, so other people can use them

Reply:
If this is the stripe index then let's call it `Index`. Then the caller of `NextStripe()` will access it with `stripe.Index`.